bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
         $ref: '#/definitions/PriceDto'
   ActivityDto:
     properties:
-      activityType:
-        $ref: '#/definitions/ActivityType'
-        description: 'Specifies the type of the activity.'
-      privateActivity:
-        type: boolean
-      bookingType:
-        $ref: '#/definitions/ActivityBookingType'
-        description: 'Specifies how the activity is booked: as a pass, date based, or date and time based.'
-      scheduleType:
-        $ref: '#/definitions/ActivityScheduleType'
-        description: 'Specifies how activity start times are scheduled. Only used if bookingType != PASS.'
-      capacityType:
-        $ref: '#/definitions/ActivityCapacityType'
-        description: 'Specifies whether capacity is limited, free sell, or on request.'
-      passExpiryType:
-        $ref: '#/definitions/PassExpiryType'
-        description: 'Only used if bookingType == PASS. Specifies how the pass expires after purchase.'
-      meetingType:
-        $ref: '#/definitions/ActivityMeetingType'
-        description: 'Describes how passengers meet at the beginning of a tour/activity.'
-      passCapacity:
+      id:
```

Comment:
I think it's worth adding `required` parameter - this mentions which elements are mandatory and which are not.

Reply:
Which place are you referring to?